lib/laminate/components/modal/modal.dart

```diff
@@ -16,7 +16,7 @@ import 'package:angular_components/utils/disposer/disposer.dart';
 ///
 /// Creates a stack of modals, hiding the previous one when a new is shown.
 ///
-/// __Example use__:
+/// __Example use:__
 ///     bootstrap(MyApp, const [GlobalModalStack]);
```

Comment:
It is recommended to use code fences rather than indentation: \```dart ... ``` (both here and at line 110 below)